GNUmakefile

```diff
@@ -2,7 +2,7 @@
 # Copyright 2021. Clumio, Inc.
 #
 
-VERSION=0.1.4
+VERSION=0.2.0
```

Comment:
We said that people can see this new version if they went to github but we're not yet releasing this new version, is that right?

Reply:
Yes.